weave/environment.py

```diff
@@ -89,6 +89,10 @@ def weave_link_prefix() -> str:
     return ""
 
 
+def weave_onprem() -> bool:
+    return os.getenv("GORILLA_ONPREM") == "true"
```

Comment:
where is `GORILLA_ONPREM` being set for the weave service?

Reply:
I just used the same env @vanpelt does for the above function:

```python
def weave_link_prefix() -> str:
    """When running in server we mount index under /weave"""
    if os.getenv("GORILLA_ONPREM") == "true":
        return "/weave"
    return ""

```

Reply:
It's set in env files in `core`.